lib/autotest/cucumber_mixin.rb

```diff
@@ -26,7 +26,7 @@ def run
 
     loop do # ^c handler
       begin
-        get_to_green
+        to_green
```

Comment:
An issue with this rename: the name is not the name of a getter, but the expression "Get to green", which is lingo from the Agile community about TDD. 

Example of this lingo in use: https://blog.carbonfive.com/2011/01/12/tdd-getting-to-green-one-small-step-at-a-time/

Reply:
I've fixed this. :)